VMBackup/main/handle.py

```diff
@@ -87,12 +88,14 @@ def main():
                 update()
             elif re.match("^([-/]*)(daemon)", a):
                 daemon()
+            elif re.match("^([-/]*)(seqNo:)", a):
+                configSeqNo = int(a.split(':')[1])
```

Comment:
I think, as a safety check, we should try to parse it as int and if that fails then reassign configSeqNo = -1